ansible/openshift-playbook.yml

```diff
@@ -11,7 +11,8 @@
     test_harness_run_as_root: true
     test_harness_node_selector:
       system-roles-ci: "true"
-    test_harness_dockerfile_from: "docker.io/library/fedora:32"
+    test_harness_dockerfile_from: "registry.fedoraproject.org/fedora:32"
+    test_harness_dockerfile_from_centos: "registry.centos.org/centos/centos:7"
```

Comment:
this should be `test_harness_dockerfile_from_centos7` to be consistent with the naming convention used by other variables which are specific to centos7 and have a `_centos7` suffix

Reply:
Fixed in #96902af